### ISSUE TITLE
Lottie json v5.7.0 issue

### ISSUE THREAD

**Issue by achrafmataich** [lottie-react-native/lottie-react-native#1133]
### Description

Hi, i just don’t know why lottie json files exported from the dashboard work fine in react native ios but not in react native android (the app crashes)

- I made the animation in fingma
- I imported it via the official lottie plugin on figma
- I exported it (download as json) today (november 20, 2023) at 00:58
- expo sdk: 49
- react native: 0.72.6
- lottie-react-native: 5.1.6
- version in the json file: 5.7.0
- last lines in the json file:
  ```json
  {
  ...
  "v":"5.7.0","fr":30,"op":81.00000321865082,"ip":0,"assets":[]
  ...
  }
  ```

I tried with the same config but only with a different lottie animation json file (that i have made 5 months ago) where the version marked is 4.8.0 and it works on both android and ios

Is that an issue with the lottie-react-native library? or maybe the last version of json files that the lottiefiles build is just bad and need to get fixed by lottie?

### Steps to Reproduce

1. Export Lottie animation as JSON
2. Put the exported JSON file in the assets of the app
3. Install the wrapper via `npm install lottie-react-native@5.1.6`
4. use it in the component like so:
  ```tsx
  //imports..
  import AnimatedLottieView from "lottie-react-native";
  const MyComponent = () => {
    //states and stuffs
    return <View>
      <AnimatedLottieView
      autoPlay
      {/*some props*/}
      source="../assets/myanimation.json"
      />
    </View>;
  }
  ```
4. Run the expo app via `npx expo start` and test it in `Expo Go`

**Expected behavior:** The animation works properly on both IOS and Android devices.

**Actual behavior:** The animation works properly on IOS but let the app crash on Android.

### Minimal reproduction

https://github.com/achrafmataich/myapp
> Here i have tested with an animation json file v4.8.0 that works fine, try to use the animation of v5.7.0 located in assets folder (the one i exported from lottie).

### React Native Environment

System:
  OS: Linux 6.2 Ubuntu 22.04.3 LTS 22.04.3 LTS (Jammy Jellyfish)
  CPU: (8) x64 Intel(R) Core(TM) i5-8300H CPU @ 2.30GHz
  Memory: 6.22 GB / 11.56 GB
  Shell:
    version: 5.1.16
    path: /bin/bash
Binaries:
  Node:
    version: 20.9.0
    path: /usr/bin/node
  Yarn: Not Found
  npm:
    version: 10.1.0
    path: /usr/bin/npm
  Watchman: Not Found
SDKs:
  Android SDK: Not Found
IDEs:
  Android Studio: Not Found
Languages:
  Java:
    version: 19.0.2
    path: /usr/bin/javac
  Ruby: Not Found
npmPackages:
  "@react-native-community/cli": Not Found
  react:
    installed: 18.2.0
    wanted: 18.2.0
  react-native:
    installed: 0.72.6
    wanted: 0.72.6
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: Not found
  newArchEnabled: Not found
iOS:
  hermesEnabled: Not found
  newArchEnabled: Not found

### Lottie Version

Version: 5.7.0

**Comment by PierreHenri123** [comment]
I have exactly the same issue !

**Comment by TheRogue76** [comment]
Hi @achrafmataich and @PierreHenri123 .This is Interesting. Can you guys send logcats from when the app crashes?

**Comment by aliunco** [comment]
@TheRogue76 Thanks for your response, I'm facing the same issue with latest version of Lottie wrapper (version 6.5.0)

Here is the json of my file:

```json
{"nm":"Start animation","ddd":0,"h":355,"w":208,"meta":{"g":"LottieFiles Figma v44"},"layers":[{"ty":4,"nm":"Sand top","sr":1,"st":0,"op":19,"ip":0,"hd":false,"ddd":0,"bm":0,"hasMask":false,"ao":0,"ks":{"a":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[98.87,157.55],"t":0},{"s":[98.87,157.55],"t":18}]},"s":{"a":0,"k":[100,100]},"sk":{"a":0,"k":0},"p":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[103.5,177.5],"t":0},{"s":[104,177.5],"t":18}]},"r":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[0],"t":0},{"s":[0],"t":18}]},"sa":{"a":0,"k":0},"o":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100],"t":0},{"s":[100],"t":18}]}},"shapes":[],"ind":1},{"ty":4,"nm":"Vector Mask","sr":1,"st":0,"op":19,"ip":0,"hd":false,"ddd":0,"bm":0,"hasMask":false,"td":1,"ao":0,"ks":{"a":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[98.87,157.55],"t":0},{"s":[98.87,157.55],"t":18.000000715255737}]},"s":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100,100],"t":0},{"s":[100,100],"t":18.000000715255737}]},"sk":{"a":0,"k":0},"p":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[98.87,157.55],"t":0},{"s":[98.87,157.55],"t":18.000000715255737}]},"r":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[0],"t":0},{"s":[0],"t":18.000000715255737}]},"sa":{"a":0,"k":0},"o":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100],"t":0},{"s":[100],"t":18.000000715255737}]}},"shapes":[{"ty":"sh","bm":0,"hd":false,"nm":"","d":1,"ks":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[{"c":true,"i":[[0,0],[0,117.75],[0,0],[0,0],[0,0],[0,-26.96],[0,0],[0,0],[0,0]],"o":[[0,-26.96],[0,0],[0,0],[0,0],[0,117.75],[0,0],[0,0],[0,0],[0,0]],"v":[[103.52,157.52],[197.74,20.8],[197.74,0],[0,0],[0,20.8],[94.23,157.52],[94.23,315.1],[103.52,315.1],[103.52,157.52]]}],"t":0},{"s":[{"c":true,"i":[[0,0],[0,117.75],[0,0],[0,0],[0,0],[0,-26.96],[0,0],[0,0],[0,0]],"o":[[0,-26.96],[0,0],[0,0],[0,0],[0,117.75],[0,0],[0,0],[0,0],[0,0]],"v":[[103.52,157.52],[197.74,20.8],[197.74,0],[0,0],[0,20.8],[94.23,157.52],[94.23,315.1],[103.52,315.1],[103.52,157.52]]}],"t":18.000000715255737}]}},{"ty":"fl","bm":0,"hd":false,"nm":"","c":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[0.851,0.851,0.851],"t":0},{"s":[0.851,0.851,0.851],"t":18.000000715255737}]},"r":1,"o":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100],"t":0},{"s":[100],"t":18.000000715255737}]}}],"ind":2,"parent":1},{"ty":4,"nm":"Sand top","sr":1,"st":0,"op":19,"ip":0,"hd":false,"ddd":0,"bm":0,"tt":1,"hasMask":false,"ao":0,"ks":{"a":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[136.94,60.12],"t":0},{"s":[136.94,138.98],"t":18.000000715255737}]},"s":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100,100],"t":0},{"s":[100,100],"t":18.000000715255737}]},"sk":{"a":0,"k":0},"p":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[104.21,97.59],"t":0},{"s":[104.21,176.45],"t":18.000000715255737}]},"r":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[0],"t":0},{"s":[0],"t":18.000000715255737}]},"sa":{"a":0,"k":0},"o":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100],"t":0},{"s":[100],"t":18.000000715255737}]}},"shapes":[{"ty":"sh","bm":0,"hd":false,"nm":"","d":1,"ks":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[{"c":true,"i":[[0,0],[0,0],[0,0],[0,0]],"o":[[0,0],[0,0],[0,0],[0,0]],"v":[[0,0],[273.87,0],[273.87,120.24],[0,120.24]]}],"t":0},{"s":[{"c":true,"i":[[0,0],[0,0],[0,0],[0,0]],"o":[[0,0],[0,0],[0,0],[0,0]],"v":[[0,0],[273.87,0],[273.87,277.96],[0,277.96]]}],"t":18.000000715255737}]}},{"ty":"fl","bm":0,"hd":false,"nm":"","c":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[0.4079,0.5451,0.9334],"t":0},{"s":[0.4079,0.5451,0.9334],"t":18.000000715255737}]},"r":1,"o":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100],"t":0},{"s":[100],"t":18.000000715255737}]}}],"ind":3,"parent":1},{"ty":4,"nm":"Sand bottom","sr":1,"st":0,"op":19,"ip":0,"hd":false,"ddd":0,"bm":0,"hasMask":false,"ao":0,"ks":{"a":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[98.87,157.52],"t":0},{"s":[98.87,157.52],"t":18}]},"s":{"a":0,"k":[100,100]},"sk":{"a":0,"k":0},"p":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[103.5,177.5],"t":0},{"s":[104,177.5],"t":18}]},"r":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[0],"t":0},{"s":[0],"t":18}]},"sa":{"a":0,"k":0},"o":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100],"t":0},{"s":[100],"t":18}]}},"shapes":[],"ind":4},{"ty":4,"nm":"Vector Mask","sr":1,"st":0,"op":19,"ip":0,"hd":false,"ddd":0,"bm":0,"hasMask":false,"td":1,"ao":0,"ks":{"a":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[98.87,157.52],"t":0},{"s":[98.87,157.52],"t":18.000000715255737}]},"s":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100,100],"t":0},{"s":[100,100],"t":18.000000715255737}]},"sk":{"a":0,"k":0},"p":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[98.87,157.52],"t":0},{"s":[98.87,157.52],"t":18.000000715255737}]},"r":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[0],"t":0},{"s":[0],"t":18.000000715255737}]},"sa":{"a":0,"k":0},"o":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100],"t":0},{"s":[100],"t":18.000000715255737}]}},"shapes":[{"ty":"sh","bm":0,"hd":false,"nm":"","d":1,"ks":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[{"c":false,"i":[[0,0],[0,117.75],[0,0],[0,0],[0,0],[0,-26.96],[0,-117.75],[0,0],[0,0],[0,0],[0,26.96]],"o":[[0,-26.96],[0,0],[0,0],[0,0],[0,117.75],[0,26.96],[0,0],[0,0],[0,0],[0,-117.75],[0,0]],"v":[[94.23,157.52],[0,20.8],[0,0],[197.74,0],[197.74,20.8],[103.52,157.52],[197.74,294.24],[197.74,315.04],[0,315.01],[0,294.21],[94.23,157.49]]}],"t":0},{"s":[{"c":false,"i":[[0,0],[0,117.75],[0,0],[0,0],[0,0],[0,-26.96],[0,-117.75],[0,0],[0,0],[0,0],[0,26.96]],"o":[[0,-26.96],[0,0],[0,0],[0,0],[0,117.75],[0,26.96],[0,0],[0,0],[0,0],[0,-117.75],[0,0]],"v":[[94.23,157.52],[0,20.8],[0,0],[197.74,0],[197.74,20.8],[103.52,157.52],[197.74,294.24],[197.74,315.04],[0,315.01],[0,294.21],[94.23,157.49]]}],"t":18.000000715255737}]}},{"ty":"fl","bm":0,"hd":false,"nm":"","c":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[0.8,0.8549,1],"t":0},{"s":[0.8,0.8549,1],"t":18.000000715255737}]},"r":1,"o":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100],"t":0},{"s":[100],"t":18.000000715255737}]}}],"ind":5,"parent":4},{"ty":4,"nm":"Polygon","sr":1,"st":0,"op":19,"ip":0,"hd":false,"ddd":0,"bm":0,"tt":1,"hasMask":false,"ao":0,"ks":{"a":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[98.88,80.67],"t":0},{"s":[98.88,80.67],"t":18.000000715255737}]},"s":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100,100],"t":0},{"s":[100,100],"t":18.000000715255737}]},"sk":{"a":0,"k":0},"p":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[98.88,398.33],"t":0},{"s":[98.88,398.33],"t":18.000000715255737}]},"r":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[0],"t":0},{"s":[0],"t":18.000000715255737}]},"sa":{"a":0,"k":0},"o":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100],"t":0},{"s":[100],"t":18.000000715255737}]}},"shapes":[{"ty":"sh","bm":0,"hd":false,"nm":"","d":1,"ks":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[{"c":true,"i":[[0,0],[0,0],[0,0],[0,0],[0,0],[0,0]],"o":[[0,0],[0,0],[0,0],[0,0],[0,0],[0,0]],"v":[[98.87,0],[197.75,83.69],[197.75,161.34],[0,161.34],[0,83.69],[98.87,0]]}],"t":0},{"s":[{"c":true,"i":[[0,0],[0,0],[0,0],[0,0],[0,0],[0,0]],"o":[[0,0],[0,0],[0,0],[0,0],[0,0],[0,0]],"v":[[98.87,0],[197.75,83.69],[197.75,161.34],[0,161.34],[0,83.69],[98.87,0]]}],"t":18.000000715255737}]}},{"ty":"fl","bm":0,"hd":false,"nm":"","c":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[0.4079,0.5451,0.9334],"t":0},{"s":[0.4079,0.5451,0.9334],"t":18.000000715255737}]},"r":1,"o":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100],"t":0},{"s":[100],"t":18.000000715255737}]}}],"ind":6,"parent":4},{"ty":4,"nm":"Top","sr":1,"st":0,"op":19,"ip":0,"hd":false,"ddd":0,"bm":0,"hasMask":false,"ao":0,"ks":{"a":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[103.5,10],"t":0},{"s":[103.5,10],"t":18}]},"s":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100,100],"t":0},{"s":[100,100],"t":18}]},"sk":{"a":0,"k":0},"p":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[103.5,10],"t":0},{"s":[103.5,10],"t":18}]},"r":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[0],"t":0},{"s":[0],"t":18}]},"sa":{"a":0,"k":0},"o":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100],"t":0},{"s":[100],"t":18}]}},"shapes":[{"ty":"sh","bm":0,"hd":false,"nm":"","d":1,"ks":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[{"c":true,"i":[[0,0],[-2.21,0],[0,0],[0,-2.21],[0,0],[2.21,0],[0,0],[0,2.21]],"o":[[0,-2.21],[0,0],[2.21,0],[0,0],[0,2.21],[0,0],[-2.21,0],[0,0]],"v":[[0,4],[4,0],[203.01,0],[207.01,4],[207.01,16],[203.01,20],[4,20],[0,16]]}],"t":0},{"s":[{"c":true,"i":[[0,0],[-2.21,0],[0,0],[0,-2.21],[0,0],[2.21,0],[0,0],[0,2.21]],"o":[[0,-2.21],[0,0],[2.21,0],[0,0],[0,2.21],[0,0],[-2.21,0],[0,0]],"v":[[0,4],[4,0],[203.01,0],[207.01,4],[207.01,16],[203.01,20],[4,20],[0,16]]}],"t":18}]}},{"ty":"fl","bm":0,"hd":false,"nm":"","c":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[0.0706,0.2745,0.8549],"t":0},{"s":[0.0706,0.2745,0.8549],"t":18}]},"r":1,"o":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100],"t":0},{"s":[100],"t":18}]}}],"ind":7},{"ty":4,"nm":"Hourglass","sr":1,"st":0,"op":19,"ip":0,"hd":false,"ddd":0,"bm":0,"hasMask":false,"ao":0,"ks":{"a":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[98.87,157.52],"t":0},{"s":[98.87,157.52],"t":18}]},"s":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100,100],"t":0},{"s":[100,100],"t":18}]},"sk":{"a":0,"k":0},"p":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[103.5,177.5],"t":0},{"s":[104,177.5],"t":18}]},"r":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[0],"t":0},{"s":[0],"t":18}]},"sa":{"a":0,"k":0},"o":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100],"t":0},{"s":[100],"t":18}]}},"shapes":[{"ty":"sh","bm":0,"hd":false,"nm":"","d":1,"ks":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[{"c":false,"i":[[0,0],[0,117.75],[0,0],[0,0],[0,0],[0,-26.96],[0,-117.75],[0,0],[0,0],[0,0],[0,26.96]],"o":[[0,-26.96],[0,0],[0,0],[0,0],[0,117.75],[0,26.96],[0,0],[0,0],[0,0],[0,-117.75],[0,0]],"v":[[94.23,157.52],[0,20.8],[0,0],[197.74,0],[197.74,20.8],[103.52,157.52],[197.74,294.24],[197.74,315.04],[0,315.01],[0,294.21],[94.23,157.49]]}],"t":0},{"s":[{"c":false,"i":[[0,0],[0,117.75],[0,0],[0,0],[0,0],[0,-26.96],[0,-117.75],[0,0],[0,0],[0,0],[0,26.96]],"o":[[0,-26.96],[0,0],[0,0],[0,0],[0,117.75],[0,26.96],[0,0],[0,0],[0,0],[0,-117.75],[0,0]],"v":[[94.23,157.52],[0,20.8],[0,0],[197.74,0],[197.74,20.8],[103.52,157.52],[197.74,294.24],[197.74,315.04],[0,315.01],[0,294.21],[94.23,157.49]]}],"t":18}]}},{"ty":"fl","bm":0,"hd":false,"nm":"","c":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[0.8,0.8549,1],"t":0},{"s":[0.8,0.8549,1],"t":18}]},"r":1,"o":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100],"t":0},{"s":[100],"t":18}]}}],"ind":8},{"ty":4,"nm":"Bottom","sr":1,"st":0,"op":19,"ip":0,"hd":false,"ddd":0,"bm":0,"hasMask":false,"ao":0,"ks":{"a":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[103.5,10],"t":0},{"s":[103.5,10],"t":18}]},"s":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100,100],"t":0},{"s":[100,100],"t":18}]},"sk":{"a":0,"k":0},"p":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[103.5,345],"t":0},{"s":[103.5,345],"t":18}]},"r":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[0],"t":0},{"s":[0],"t":18}]},"sa":{"a":0,"k":0},"o":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100],"t":0},{"s":[100],"t":18}]}},"shapes":[{"ty":"sh","bm":0,"hd":false,"nm":"","d":1,"ks":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[{"c":true,"i":[[0,0],[-2.21,0],[0,0],[0,-2.21],[0,0],[2.21,0],[0,0],[0,2.21]],"o":[[0,-2.21],[0,0],[2.21,0],[0,0],[0,2.21],[0,0],[-2.21,0],[0,0]],"v":[[0,4],[4,0],[203.01,0],[207.01,4],[207.01,16],[203.01,20],[4,20],[0,16]]}],"t":0},{"s":[{"c":true,"i":[[0,0],[-2.21,0],[0,0],[0,-2.21],[0,0],[2.21,0],[0,0],[0,2.21]],"o":[[0,-2.21],[0,0],[2.21,0],[0,0],[0,2.21],[0,0],[-2.21,0],[0,0]],"v":[[0,4],[4,0],[203.01,0],[207.01,4],[207.01,16],[203.01,20],[4,20],[0,16]]}],"t":18}]}},{"ty":"fl","bm":0,"hd":false,"nm":"","c":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[0.0706,0.2745,0.8549],"t":0},{"s":[0.0706,0.2745,0.8549],"t":18}]},"r":1,"o":{"a":1,"k":[{"o":{"x":0.32,"y":0},"i":{"x":0.67,"y":0},"s":[100],"t":0},{"s":[100],"t":18}]}}],"ind":9}],"v":"5.7.0","fr":30,"op":18,"ip":0}
```

Unfortunately there's nothing in the log! app just crashes and closes.
This issue is only available in the Android! (in iOS it's working properly)

**Comment by TheRogue76** [comment]
@aliunco Thanks for sending me the sample JSON.

I ran it on Android in release mode for both New arch and Old arch:

New arch:
<img width="1728" alt="Screenshot 2024-01-03 at 11 02 07" src="https://github.com/lottie-react-native/lottie-react-native/assets/40071952/b5b13fe6-dd74-4c7f-83c3-c5748e76e963">

Old arch:
<img width="1728" alt="Screenshot 2024-01-03 at 11 14 51" src="https://github.com/lottie-react-native/lottie-react-native/assets/40071952/bd9f55d1-b63c-424e-90b7-9db1d0b195ae">

And i could not get it to crash or even show me an error. Which devices are you running into this issue with?

**Comment by TheRogue76** [comment]
Same with yours @achrafmataich

Old arch:
<img width="1728" alt="Screenshot 2024-01-03 at 11 23 35" src="https://github.com/lottie-react-native/lottie-react-native/assets/40071952/ed53d4e2-c7a1-45bf-b3fc-ab344029817f">

New arch:
<img width="1728" alt="Screenshot 2024-01-03 at 11 25 48" src="https://github.com/lottie-react-native/lottie-react-native/assets/40071952/db4943f5-0b45-47b4-8f38-40aaa7994152">

**Comment by aliunco** [comment]
@TheRogue76 Thank you so much for this quick response! 
I'm using an actual device (Samsung S21 Ultra 5G) using Expo go!

**Comment by TheRogue76** [comment]
> @TheRogue76 Thank you so much for this quick response! I'm using a real device (Samsung S21 Ultra 5G) using Expo go!
> 
> I can also inform you that I tried with a raw (new) Expo project! and I could reproduce the crash!

Expo Go is a client created by the Expo team that uses a hardcoded version of Lottie (You can see the version info [here](https://github.com/expo/expo/blob/e9e61125dcbfeac7efe5af1ef9dfd2792c195cdf/apps/native-component-list/package.json#L128)). It doesn't matter what your version of Lottie is in your project, the Expo Go client is bound to whatever version they created it with.

There also might be other things that are wrong with Expo go. Without more logs (The Logcat from the Android device in question, which you can read more about [here](https://developer.android.com/studio/debug/logcat)), no one can say for sure.

My recommendation is to open an issue on their repository. Perhaps they can tell you if that is a known issue.

**Comment by TheRogue76** [comment]
Pinging @kudo because i know he is the last person who updated Lottie on Expo go. I also tested this Lottie file against version 6.4.1, which is the version the Expo go client is using as far as i see, and it works fine on that version as well from what i see:
<img width="1728" alt="Screenshot 2024-01-04 at 04 17 57" src="https://github.com/lottie-react-native/lottie-react-native/assets/40071952/23830df2-fd9d-494c-b0d7-e236f866e75c">

**Comment by aliunco** [comment]
@TheRogue76 thank you so much for your time! I'll try to investigate more on my machine to find the issue (maybe there's something specific on my device, cause I could reproduce it with latest versions of expo go and Lottie)

```json
{
  "name": "testapp",
  "version": "1.0.0",
  "main": "node_modules/expo/AppEntry.js",
  "scripts": {
    "start": "expo start",
    "android": "expo start --android",
    "ios": "expo start --ios",
    "web": "expo start --web"
  },
  "dependencies": {
    "expo": "~49.0.15",
    "expo-status-bar": "~1.6.0",
    "lottie-react-native": "^6.5.0",
    "react": "18.2.0",
    "react-native": "0.72.6"
  },
  "devDependencies": {
    "@babel/core": "^7.20.0"
  },
  "private": true
}
```

same behavior! 
node version v16.19.0
OS: Sonoma 14.2
Expo go client version: 2.29.8

**Comment by Kudo** [comment]
hi there! this is kudo from expo. expo-go 2.29.8 is still using lottie-react-native 5.1.6. to try out latest lottie-react-native, you could either build your own [development build](https://docs.expo.dev/develop/development-builds/introduction/), or staying tune., when expo sdk 50 officially released this month, we will publish expo-go 2.30 to google play store. expo-go 2.30 should at least has lottie-react-native 6.4.1

**Comment by TheRogue76** [comment]
Closing this issue as it is no longer relevant to this repo. I will not lock the conversation so feel free to keep talking logistics.
